cmd/generate/config/utils/generate.go

```diff
@@ -18,16 +18,17 @@ const (
 	identifierCaseInsensitivePrefix = `[\w.-]{0,50}?(?i:`
 	identifierCaseInsensitiveSuffix = `)`
 	identifierPrefix                = `[\w.-]{0,50}?(?:`
-	identifierSuffix                = `)(?:[ \t\w.-]{0,20})(?:[\s|']|[\s|"]){0,3}`
+	identifierSuffix                = `)(?:[\w \t.-]{0,20})(?:\\*[\'\"]{0,3})?[ \t]{0,5}`
```

Comment:
@rgmz could you elaborate on this change? I'm worried removing `(?:[\s|']|[\s|"]){0,3}` might cause some TPs to slip through